asmcli/commands/help.sh

```diff
@@ -249,3 +249,52 @@ FLAGS:
   --version
 EOF
 }
+
+create-mesh_usage() {
+  cat << EOF
+${SCRIPT_NAME} $(version_message)
+usage: ${SCRIPT_NAME} create-mesh FLEET_ID (PROJECT_ID/CLUSTER_LOCATION/CLUSTER_NAME | KUBECONFIG_PATH) ... 
+
+Create a service mesh using the GKE clusters provided by registering them to the fleet and install remote secrets on each
+pair of clusters.
+Cluster information and KUBECONFIG could be passed in interchangeably.
+You could pass in as many clusters as you want, as long as they are not already registered to other fleets.
+Clusters already registered to the specified fleet will be skipped, whereas those registered to other fleets will fail.
+
+FLAGS:
+  The following several flags are used to display help texts and the version message. 
+  -v|--verbose                        Print commands before and after execution.
+  -h|--help                           Show this message and exit.
+  --version                           Print the version of this tool and exit.
+
+EXAMPLE:
+The following invocation will create a mesh using the provided 6 clusters 
+by registering them to a fleet named "my_cluster" and install remote secrets on each pair of clusters:
+
+  $> ${SCRIPT_NAME} \\
+      create-mesh \\
+      my-fleet \\
+      my-project/us-central1-c/my-cluster1 \\
+      my-project/us-central1-c/my-cluster2 \\
+      my-project/us-central1-c/my-cluster3 \\
+      path/to/kubeconfig1 \\
+      path/to/kubeconfig2 \\ 
+      path/to/kubeconfig3 \\
```

Comment:
I think we could probably just get away with

```
my-project/us-central1-c/my-cluster1 \\
path/to/kubeconfig2 \\ 
my-project/us-central1-c/my-cluster3
```

(Also note that the command would fail if you had trailing `\\` on the last line.)

Reply:
Done